### PR TITLE
My first pull Request. 

### PR DESCRIPTION
**Pull request by paulgureghian**
Creating a pull request to submit to Microverse.

### CODE REVIEW

**Review by uche-inyama — COMMENTED** [review]
## Pull request under review by Inyama, Uchechukwu Henry.

**Review by uche-inyama — CHANGES_REQUESTED** [review]
## Hi Paul :wave: you have done a good job.

### However, there are changes that needs to be implemented.


- [x] Validate you html code, use this [link](https://validator.w3.org). Ensure there are no errors.
- [x] Use the **```<main> tag```**  to wrap the main content of your html document.

- [x] Make your README.md file descriptive. Here are some pointers :dart: 

     *  Write down the technologies used in your project.
     * The team that worked on the project, with the link to their github account.
     * A link to Odin Project, for more details.
    * A link to live version of your clone, you can get a link from [Githack](https://raw.githack.com/)


- [x] Change the color of your navigation bar, it should not be black. Use the snap shot below as a reference :point_down: 

Screenshot from 2019-09-19 20-30-23

### After implementing these changes, submit another code review form. :muscle: 

### This project is not approved :stop_sign:

**Comment by mkezfarias** [comment]
Fixed issues and resubmitted

**Comment by wathigo** [comment]
**Reviewing...**

**Review by wathigo — CHANGES_REQUESTED** [review]
## Hi Devs :wave: 
### Great work! The clone looks great!
Please let's fix this before moving on:
* Your CSS code contains some errors. You have fixed all the errors for html code but there still error in your CSS. Upload your code to [w3 validator](https://validator.w3.org/#validate_by_upload) and correct all the errors. 
* Unimportant files such as `.DS_Store` should not be commited. Include these file in `.gitignore` file.

**After implementing the changes, submit another request for review.**
_Good luck :)_

**Comment by mkezfarias** [comment]
fixed css , add gitignore file and resubmitted for review

File: styles.css [hunk 1]
```diff
@@ -1,77 +1,73 @@
...
+  border-top: 10px #EEEEEE;
+  padding: 30px;
 }
 
-#channel_info{
-  display:grid;
+.channel_info {
+  display: grid;
   grid-template-columns: 3 auto;
   grid-template-rows: 2 auto;
```
Comment (samgaco) [inline styles.css]: You have to put a unit after 3 and 2 (px, rm, etc.)

File: styles.css [hunk 2]
```diff
@@ -107,159 +105,117 @@ body {
...
+  font-weight: lighter;
+  margin-top: 15px;
 }
 
-#subscriber_count {
+.subscriber_count {
   color: gray;
-  size: 12px;
   padding: 0;
+  size: 12px;
```
Comment (samgaco) [inline styles.css]: > Property size doesn't exist. The closest matching property name is resize : 12px

**Review by samgaco — APPROVED** [review]
HI @paulgureghian and @mkezmkez :high_brightness: excellent work, it's approved :white_check_mark: you may merge to the master branch and submit the project completion form :+1: 

**Before merging,** please address the errors given by the[ CSS validator](https://jigsaw.w3.org) when uploading the style.css file.

----------------

Moving on :arrow_down: 
* **[For your next code review requests]**  Use meaningful names for your Pull Requests

--------------------
* ###### STATUS OF THE PROJECT: APPROVED :heavy_check_mark:

File: styles.css (hunk 1 above)
Comment (paulgureghian) [reply styles.css]: I used 'px'  but I am not sure if that is the correct unit to use.

File: styles.css (hunk 2 above)
Comment (paulgureghian) [reply styles.css]: I changed it to 'resize: 12px;

File: styles.css (hunk 1 above)
Comment (paulgureghian) [reply styles.css]: I used 'px' but I am not sure that is the correct unit.

**Pull request merged**